net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -334,9 +369,26 @@ if /bin/pgrep -q ospfd; then
 	sleep 1
 	/usr/bin/killall -9 ospfd 2>/dev/null
 fi
+if /bin/pgrep -q ospf6d; then
+        /usr/bin/killall ospf6d
+        sleep 1
+        /usr/bin/killall -9 ospf6d 2>/dev/null
+fi
+if /bin/pgrep -q bgpd; then
+	/usr/bin/killall bgpd
+	sleep 1
+	/usr/bin/killall -9 bgpd 2>/dev/null
+fi
 {$carp_ip_status_check}
+/usr/bin/touch {$quagga_config_base}/zebra.conf
+/usr/bin/touch {$quagga_config_base}/bgpd.conf
+/usr/bin/touch {$quagga_config_base}/ospfd.conf
+/usr/bin/touch {$quagga_config_base}/ospf6d.conf
 /usr/local/sbin/zebra -d -f {$quagga_config_base}/zebra.conf
 /usr/local/sbin/ospfd -d -f {$quagga_config_base}/ospfd.conf
+/usr/local/sbin/ospf6d -d -f {$quagga_config_base}/ospf6d.conf
```

Comment:
Why start this unconditionally? What if its configuration is empty? It should not start if there is no configuration. At a minimum, test that the file exists and has non-zero size.